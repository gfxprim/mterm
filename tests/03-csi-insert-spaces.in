80 2
abcdefghijklmnopqrstuvwxyzhello\b\b\b\b\b\b\b\e[1@a
