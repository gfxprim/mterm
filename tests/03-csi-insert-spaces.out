 --------------------------------------------------------------------------------
|abcdefghijklmnopqrstuvwxayzhello                                                |
|                                                                                |
 --------------------------------------------------------------------------------
size 2x80 cursor 0x25
